storage/database/db_migration.go

```diff
@@ -75,29 +57,113 @@ loop:
 			return errors.Wrap(err, "failed to put batch")
 		}
 
+		if dstBatch.ValueSize() > IdealBatchSize {
+			if err := dstBatch.Write(); err != nil {
+				return err
+			}
+			dstBatch.Reset()
+		}
+
 		// make a report
-		if fetched%reportCycle == 0 {
+		if fetched%(IdealBatchSize*20) == 0 {
 			logger.Info("DB migrated",
-				"fetched", fetched-previousFetched, "elapsedIter", time.Since(cycleStart),
-				"fetchedTotal", fetched, "elapsedTotal", time.Since(start))
-			cycleStart = time.Now()
-			previousFetched = fetched
+				"db", name, "fetchedTotal", fetched, "elapsedTotal", time.Since(start))
 		}
 
 		// check for quit signal from OS
 		select {
-		case <-sigQuit:
-			logger.Info("exit called", "fetchedTotal", fetched, "elapsedTotal", time.Since(start))
-			break loop
+		case <-quit:
+			logger.Warn("exit called", "db", name, "fetchedTotal", fetched, "elapsedTotal", time.Since(start))
+			return nil
 		default:
 		}
 	}
 
 	if err := dstBatch.Write(); err != nil {
 		return errors.Wrap(err, "failed to write items")
 	}
+	dstBatch.Reset()
+
+	logger.Info("Finish DB migration", "db", name, "fetchedTotal", fetched, "elapsedTotal", time.Since(start))
+
+	srcIter.Release()
+	if err := srcIter.Error(); err != nil { // any accumulated error from iterator
+		return errors.Wrap(err, "failed to iterate")
```

Comment:
`errors.Wrap` returns an error annotating err with a stack trace.
Do we need stack trace here?

```
// Wrap returns an error annotating err with a stack trace
// at the point Wrap is called, and the supplied message.
// If err is nil, Wrap returns nil.
func Wrap(err error, message string) error {
	if err == nil {
		return nil
	}
	err = &withMessage{
		cause: err,
		msg:   message,
	}
	return &withStack{
		err,
		callers(),
	}
}

```

Reply:
The stack trace is returned when `+v` is used in format. It is not returned in Klaytn log.
`errors.Wrap` can be replaced with `errors.WithMessage` which is `errors.Wrap` without the stack trace.

Reply:
@winnie-byun If we don't print out stack trace in Klaytn, I think there's no need to use `errors.Wrap` instead of `erros.WithMessage`. How do you think?

Reply:
I think both ways are fine :)

Reply:
```suggestion
		return errors.withMessage(err, "failed to iterate")
```